sos/plugins/kubernetes.py

```diff
@@ -22,33 +23,101 @@ class kubernetes(Plugin, RedHatPlugin):
     """Kubernetes plugin
     """
 
-    option_list = [("podslog", "capture logs for pods", 'slow', False)]
+    # Red Hat Atomic Platform and OpenShift Enterprise use the
+    # atomic-openshift-master package to provide kubernetes
+    packages = ('kubernetes', 'atomic-openshift-master')
```

Comment:
I believe I have a fair understanding of how `--sysroot` works, but I fail to see how it could help here in any practical way.
Just to clarify: I'm talking about an installation where the kubernetes master is running like this:
```
-bash-4.2# ps ax | grep atomic-openshift-master
  2023 ?        Ssl    0:00 /usr/bin/docker run --rm --privileged --net=host --name atomic-openshift-master -v /var/lib/origin:/var/lib/origin -v /var/run/docker.sock:/var/run/docker.sock -v /etc/origin:/etc/origin openshift3/ose start master --config=/etc/origin/master/master-config.yaml --loglevel=2
```
The `atomic-openshift-master` package was installed inside the image where this container is running from (`openshift3/ose`). You are running `sosreport` from another container. Even if you were able to chroot into the ose container (which is not that clear), you probably don't want to because you most likely want sos to collect information about the host... i.e. you'd be running sos with a chroot to /host where the host's root filesystem is mounted, right? and the atomic-openshift-master package is not in the rpm db there.